Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -6699,7 +6699,7 @@ WebInspectorEngineeringSettingsAllowed:
 
 WebLocksAPIEnabled:
   type: bool
-  status: developer
+  status: stable
```

Comment:
Shouldn't this be "shipping" ?

Reply:
This is also relatively new, so we should keep the toggle until next major release.